### DISCUSSION TITLE
Support for .msh mesh files

### DISCUSSION THREAD

**Discussion by hugojmaia** [cnr-isti-vclab/meshlab#1529] [Ideas]
All the info I've found on the file format is that it's tied up with 3DS Max, which is an Autodesk tool, so running it on linux becomes a no-go.
I'm looking to work on the 3D model for Grim Dawn and they all seem tied up in that format.
Would be nice to be able to to use something non-proprietary to work on them.
Here's one of the files in question.
[aetherialobelisk01.msh.zip](https://github.com/user-attachments/files/16980602/aetherialobelisk01.msh.zip)

A tool that can convert the msh to and from a file format that Meshlab can work on would also solve this.

**Comment by alemuntoni** [comment]
Do you mean the [gmsh](https://gmsh.info/doc/texinfo/gmsh.html) format? It does not seem related to 3ds max. 
I did a quick search and I did not found any C++ open source implementation of an importer/exporter...
Anyway, PRs are always welcome!!

**Comment by hugojmaia** [comment]
Grim Dawn provides the plugins for it with the game install, so they're all proprietary after all.
I don't have the know how to contribute with this, but thanks for taking a look.
